naga/src/valid/analyzer.rs

```diff
@@ -1113,6 +1147,34 @@ impl FunctionInfo {
                     }
                     FunctionUniformity::new()
                 }
+                S::MeshFunction(func) => match &func {
+                    // TODO: double check all of this uniformity stuff. I frankly don't fully understand all of it.
+                    &crate::MeshFunction::SetMeshOutputs {
```

Comment:
I don't see anything that would prevent this function from being called outside a mesh shader and its callees. That should be an error, right?